Designs/create_shoppinglist.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Create BucketList</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,300,100" rel="stylesheet">
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+
+<form class="card" id="new_bkt">
+    <div class="form-header">
+        <h4 class="title">Create A new Shopping list</h4>
+    </div>
+    <div class="container">
+        <label><b>Name</b></label>
+        <input type="text" placeholder="Enter ShoppingList Name" name="shopping_list_name" required>
```

Comment:
Same comment as above. Strive to have every user-facing text on your app as grammatically correct as possible.